.github/workflows/golang-security-scan.yml

```diff
@@ -266,6 +278,7 @@ jobs:
     name: Create issues for failed jobs
     runs-on: ubuntu-latest
     strategy:
+      fail-fast: false
```

Comment:
Could you elaborate on why this is a helpful change?

Reply:
We're using a matrix strategy here, so each matrix label runs separately in parallel (see attached image). The default behavior is `fail-fast: true`, which means if one of the matrix runs fails then the others will be canceled. However, if one fails for some reason, we would prefer that the others still continue to run.

![image](https://github.com/cccteam/github-workflows/assets/131682132/45fe5091-3608-474a-a59b-6ae46ed17f49)